source/config-file/config-file-options.txt

```diff
@@ -68,12 +68,18 @@ To view a sample |compass-short| configuration file, run the following command:
      <path-to-Compass-executable> --show-example-config
 
 If you do not already have a configuration file, you can use the following 
-command to run |compass-short| with the sample configuration file: 
+command to create a configuration file that uses the sample configuration 
```

Comment:
Consider removing "following"

... you can create a configuration file that uses the sample ...

Reply:
Noted and updated!